src/components/Counter/ReportInfoButton/ReportInfoButton.js

```diff
@@ -38,7 +38,9 @@ const ReportInfoButton = ({
   };
 
   const getButtonIcon = (failedAttempts) => {
```

Comment:
Please update the method signature to include all used parameters, or alternatively, none. The current method signature for `getButtonIcon` suggests that it only depends on the `failedAttempts` parameter. However, the function actually relies on multiple properties: `report.failedAttempts`, `report.reportEditedManually`, and `maxFailedAttempts`.

Reply:
You are right! I removed the parameter.